Sentinel/Classes/UserDefaults/UserDefaultsToolDetailView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UserDefaultsToolDetailView.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 28.11.2024..
+//
+
+import SwiftUI
+
+struct UserDefaultsToolDetailView: View {
+
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    @ObservedObject var viewModel: UserDefaultsToolDetailViewModel
+    #if os(macOS)
+    @Binding var selection: String?
```

Comment:
Can we move this to the `viewModel`?

Reply:
Since it's a view specific binding, I feel like we should keep it here